spring-boot-modules/spring-boot-basic-customization-3/src/test/java/com/baeldung/responsebody/ResponseBodyUnitTest.java

```diff
@@ -0,0 +1,48 @@
+package com.baeldung.responsebody;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import jakarta.xml.bind.DatatypeConverter;
+
+@SpringBootTest(classes = ResponseBodyApplication.class)
+@AutoConfigureMockMvc
+class ResponseBodyUnitTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    void testMD5FilterForGetRequest() throws Exception {
```

Comment:
Please name the test following the convention: givenXX_whenXXX_thenXXX

Reply:
Renamed it.